### PR TITLE
Ropes Issue 5: Image Tags

### PR DESCRIPTION
**Pull request by modethirteen**
See https://trello.com/c/z3WngaVk/396-issue-5-image-tags

### CODE REVIEW

**Comment by modethirteen** [comment]
[Issue 5: Image Tags](https://trello.com/c/z3WngaVk/396-issue-5-image-tags)

File: db/seeds.rb [hunk 1]
```diff
@@ -6,5 +6,9 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 (20..39).each do |id|
-  Image.create(url: "https://picsum.photos/id/#{id}/400/400.jpg")
+  image = Image.create(url: "https://picsum.photos/id/#{id}/400/400.jpg")
+  %w[foo bar baz plugh xyzzy fred qux bazz foobar quxx].sample(3).each do |tag|
+    image.tag_list.add(tag)
+  end
+  image.save
```
Comment (darreneng) [inline db/seeds.rb:13]: for this I would suggest using `image.save!` so this raises an exception if it fails

File: test/models/image_test.rb [hunk 2]
```diff
@@ -94,4 +94,11 @@ class ImageTest < ActiveSupport::TestCase
     assert_same bar.previous.id, baz.id
     assert_same baz.previous.id, foo.id
   end
+
+  test 'can add tags' do
+    foo = Image.create(url: 'https://example.com/xyzzy.png')
```
Comment (darreneng) [inline test/models/image_test.rb:99]: similar to my other comments, prefer `create!` over `create`

File: db/seeds.rb (hunk 1 above)
Comment (modethirteen) [reply db/seeds.rb:13]: Ah, thank you - I was pretty annoyed by silent failures 👍

File: test/models/image_test.rb (hunk 2 above)
Comment (modethirteen) [reply test/models/image_test.rb:99]: 👍

**Pull request merged**